src/util/time_lua.inl

```diff
@@ -312,13 +363,19 @@ namespace {
 }
 
 /**
- * Modifies the passed time (first argument) by the delta time (second argument). The
- * first argument can either be an ISO 8601 date string or the number of seconds past the
- * J2000 epoch. The second argument can either be a string of the form [-]XX(s,m,h,d,M,y]
- * with (s)econds, (m)inutes, (h)ours, (d)ays, (M)onths, and (y)ears as units and an
- * optional - sign to move backwards in time. If the second argument is a number, it is
- * interpreted as a number of seconds. The return value is of the same type as the first
- * argument.
+ * Modify a specified timestamp by a given delta time. That is, advance the value either
+ * forwards or backwards in time.
+ *
+ * The returned value will be of the same type as the first argument. That is, either a
+ * number of seconds past the J2000 epoch, or an ISO 8601 date string.
+ *
+ * \param base The timestamp to alter, either given as an ISO 8601 date string or a
+ *             number of seconds past the J2000 epoch.
+ * \param change The amount of time to add to the specified timestamp. Can be given
+ *               either in a number of seconds (including negative), or as a string of
+ *               the form [-]XX(s,m,h,d,M,y] with (s)econds, (m)inutes, (h)ours, (d)ays,
+ *               (M)onths, and (y)ears as units and an optional - sign to move backwards
+ *               in time.
  */
```

Comment:
I have forgotten \return arguments for all the functions that return things... Should be fixed